ocf_data_sampler/numpy_sample/site.py

```diff
@@ -0,0 +1,29 @@
+"""Convert site to Numpy Sample"""
+
+import xarray as xr
+
+
+class SiteSampleKey:
+
+    generation = "site"
+    capacity_kwp = "site_capacity_kwp"
+    time_utc = "site_time_utc"
+    t0_idx = "site_t0_idx"
+    solar_azimuth = "site_solar_azimuth"
+    solar_elevation = "site_solar_elevation"
+    id = "site_id"
+
+
+def convert_site_to_numpy_sample(da: xr.DataArray, t0_idx: int | None = None) -> dict:
```

Comment:
Likewise. Not strictly necessary - but perhaps validation check for required input variables? Safe measure